src/components/modules/nutrition/daily-intake/DailyIntakeFoods.vue

```diff
@@ -60,19 +62,13 @@ export default class DailyIntakeFoods extends Vue {
 </script>
 
 <style lang="scss" scoped>
-    .title-container {
-        display: flex;
-        justify-content: space-between
-    }
-
-    .nutrients-container {
-        display: flex;
-        width: 100%;
-    }
+    @import '../../../../styles/list';
 
-    .nutritional-value {
-        display: flex;
-        flex-direction: column;
+    .constrained {
+        max-width: 80%;
+        white-space: nowrap;
+        overflow: hidden;
+        text-overflow: ellipsis;
```

Comment:
I see that you're repeating those three lines:
```
white-space: nowrap;
overflow: hidden;
text-overflow: ellipsis;
```

Maybe you could move them to separate class e.g. `text-truncate`?